release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3672,3 +3672,29 @@ $ oc adm release info 4.13.29 --pullspecs
 [id="ocp-4-13-29-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-30"]
+=== RHSA-2024:0286 - {product-title} 4.13.30 bug fix and security update
+
+Issued: 2024-01-24
+
+{product-title} release 4.13.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:0286[RHBA-2024:0286] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0288[RHSA-2024:0288] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.30 --pullspecs
+----
+
+[id="ocp-4-13-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, using EUS channels for mirroring releases was causing a failure in mirroring with `oc-mirror`.
+This was due to the fact that `oc-mirror` did not acknowledge that EUS channels are for even-numbered releases only. With this release, users of `oc-mirror` can now use EUS channels for mirroring releases. (link:https://issues.redhat.com/browse/OCPBUGS-26595[*OCPBUGS-26595*])
```

Comment:
```suggestion
This issue occurred because`oc-mirror` did not acknowledge that EUS channels are for even-numbered releases only. With this release, users of the `oc-mirror` command can now use EUS channels for mirroring releases. (link:https://issues.redhat.com/browse/OCPBUGS-26595[*OCPBUGS-26595*])
```